modules/os-jdg7-conffiles/module.yaml

```diff
@@ -0,0 +1,7 @@
+schema_version: 1
+name: os-jdg7-conffiles
```

Comment:
These names will need to change to avoid conflicts with the ones in `cct_module`.  If these modules are to be shared across JDG versions, it may be better to use a separate git repo for the modules.  With this change, you may need to cherry-pick patches to other version branches (just fyi for consideration).

Also, modules are intended to fully encapsulate a feature or function.  If there's no reason to split details into separate modules (i.e. you can't specify the function/feature that it provides or if there is cross/circular dependency between modules, e.g. one needs to include all or none), then I'd recommend consolidating all of this into a single module.  You might want to leave the standalone.xml stuff as a separate module as I suspect that would evolve separately from the config scripts.  The resulting bits might look something like:

image.yaml
```yaml
modules:
  repositories:
  - name: cct_module
    git:
      url: https://github.com/jboss-openshift/cct_module.git
      ref: master
  - name: jboss.datagrid.modules
    path: modules/common
  - name: jboss.datagrid.72.modules
    path: modules/72
  install:
  - jboss.datagrid.launch # common launch scripts
  - ...
```

modules/common/datagrid/launch
```yaml
modules:
  install:
  - jboss.datagrid.configuration
```
Where jboss.datagrid.configuration would be located at modules/72/datagrid/configuration, which would contain the custom standalone.xml specific to datagrid 7.2.

Other versions could use the same approach, specifying a different repo containing the configuration module, so that as long as the scripts are backward compatible, you could add support for new features (e.g. this new env) and document those as unsupported in older versions.  Obviously, if the configuration schema changes, you'd need new scripts.

Reply:
@rcernich Thanks for the feedback, all very good points. I just naively copied the existing modules for ease, but it makes sense as this point to reevaluate what individual modules we actually need and how they are implemented.  

I think we (JDG) need to discuss this in greater detail and agree upon a strategy. @slaskawi @osnipso I will put this PR on hold until our sync on Monday and we can discuss then.